Tutorial gallery is broken

Trying to build it gives this error:

```shell
(py34devel)[damian@damian-S400CA][sphinx](hotfix/tutorial)$ make clean tutorial
rm -rf _build/*
sphinx-build -b html -d _build/doctrees   tutorial_source _build/html
Making output directory...
Running Sphinx v1.2.3
loading pickled environment... failed: [Errno 2] No such file or directory: '/media/datos/Desarrollos/bokeh/sphinx/_build/doctrees/environment.pickle'
building [html]: targets for 7 source files that are out of date
updating environment: 7 added, 0 changed, 0 removed
reading sources... [100%] topical                                                                                                                                       
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/more_info.rst:4: WARNING: toctree glob pattern 'solutions/gallery/*' didn't match any documents
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] topical                                                                                                                                        
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/advanced.rst:21: WARNING: unknown document: solutions/gallery/les_mis
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/advanced.rst:32: WARNING: unknown document: solutions/gallery/periodic
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:14: WARNING: unknown document: solutions/gallery/lines
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:27: WARNING: unknown document: solutions/gallery/scatter
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:40: WARNING: unknown document: solutions/gallery/image
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:53: WARNING: unknown document: solutions/gallery/style
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:66: WARNING: unknown document: solutions/gallery/brushing
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/basic.rst:79: WARNING: unknown document: solutions/gallery/hover
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/quick_install.rst:7: WARNING: undefined label: installation (if the link has no caption the label must precede a section header)
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/topical.rst:17: WARNING: unknown document: solutions/gallery/stocks
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/topical.rst:38: WARNING: unknown document: solutions/gallery/histogram
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/topical.rst:51: WARNING: unknown document: solutions/gallery/boxplot
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/topical.rst:67: WARNING: unknown document: solutions/gallery/unemployment
/media/datos/Desarrollos/bokeh/sphinx/tutorial_source/topical.rst:84: WARNING: unknown document: solutions/gallery/olympics
writing additional files... genindex search
copying images... [100%] _images/thumbs/olympics_t.png                                                                                                                  
copying static files... WARNING: favicon file '../_static/bokeh_white_32.ico' does not exist
done
copying extra files... done
dumping search index... done
dumping object inventory... done
build succeeded, 16 warnings.

Build finished. The HTML pages are in _build/html.
cp -r _build/tutorial_gallery _build/html/tutorial/solutions/gallery
cp: no se puede efectuar `stat' sobre «_build/tutorial_gallery»: No existe el archivo o el directorio
make: *** [tutorial] Error 1
```

ping @bryevdv

--- a/sphinx/source/tutorial/solutions/.gitignore
+++ b/sphinx/source/tutorial/solutions/.gitignore
@@ -1,3 +1,2 @@
-gallery/*
 *.html
 *.json

--- a/sphinx/source/tutorial/solutions/gallery/boxplot.rst
+++ b/sphinx/source/tutorial/solutions/gallery/boxplot.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/boxplot.py

--- a/sphinx/source/tutorial/solutions/gallery/brushing.rst
+++ b/sphinx/source/tutorial/solutions/gallery/brushing.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/brushing.py

--- a/sphinx/source/tutorial/solutions/gallery/histogram.rst
+++ b/sphinx/source/tutorial/solutions/gallery/histogram.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/histogram.py

--- a/sphinx/source/tutorial/solutions/gallery/hover.rst
+++ b/sphinx/source/tutorial/solutions/gallery/hover.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/hover.py

--- a/sphinx/source/tutorial/solutions/gallery/image.rst
+++ b/sphinx/source/tutorial/solutions/gallery/image.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/image.py

--- a/sphinx/source/tutorial/solutions/gallery/les_mis.rst
+++ b/sphinx/source/tutorial/solutions/gallery/les_mis.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/les_mis.py

--- a/sphinx/source/tutorial/solutions/gallery/lines.rst
+++ b/sphinx/source/tutorial/solutions/gallery/lines.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/lines.py

--- a/sphinx/source/tutorial/solutions/gallery/olympics.rst
+++ b/sphinx/source/tutorial/solutions/gallery/olympics.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/olympics.py

--- a/sphinx/source/tutorial/solutions/gallery/periodic.rst
+++ b/sphinx/source/tutorial/solutions/gallery/periodic.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/periodic.py

--- a/sphinx/source/tutorial/solutions/gallery/scatter.rst
+++ b/sphinx/source/tutorial/solutions/gallery/scatter.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/scatter.py

--- a/sphinx/source/tutorial/solutions/gallery/stocks.rst
+++ b/sphinx/source/tutorial/solutions/gallery/stocks.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/stocks.py

--- a/sphinx/source/tutorial/solutions/gallery/style.rst
+++ b/sphinx/source/tutorial/solutions/gallery/style.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/style.py

--- a/sphinx/source/tutorial/solutions/gallery/unemployment.rst
+++ b/sphinx/source/tutorial/solutions/gallery/unemployment.rst
@@ -1,0 +1,6 @@
+
+
+solution
+########
+
+.. bokeh-plot:: source/tutorial/solutions/unemployment.py

